integrationtests/rtt_test.go

```diff
@@ -59,8 +64,22 @@ var _ = Describe("non-zero RTT", func() {
 
 		Context(fmt.Sprintf("with quic version %d", version), func() {
 			It("gets a file with 10ms RTT", func() {
-				runRTTTest(10*time.Millisecond, version)
+				runRTTTest(10*time.Millisecond, version, 20)
 			})
+
+			fileSizes := [...]int{dataLen, 5 * 1024 * 1024, dataLongLen}
+			for _, fileSize := range fileSizes {
+				fileSizeMB := fmt.Sprintf("%.2f", float64(fileSize)/1024/1024)
```

Comment:
I don't think we gain anything from running the test with various file sizes.
I'd rather run the test with different RTT values here, not just 10ms and 75ms.